config.py

```diff
@@ -102,7 +102,7 @@ def __init__(self, config_file):
                     self.dataset = config.get('dataset')
                     self.no_parents = config.get('no_parents')
                     self.split_transactions = config.get('split_transactions')
-                    self.upload_log_dir = config.get('upload_log_dir')
+                    self.upload_log_dir = config.get('upload-log-dir')
```

Comment:
Please use "**upload_log_dir**" in config file to be consistent with other variables there.

Reply:
Done the change and checked in.  Thanks!